iib/workers/tasks/build.py

```diff
@@ -808,24 +808,30 @@ def handle_add_request(
         arches = prebuild_info['arches']
         if deprecation_bundles:
             if is_fbc:
```

Comment:
I have problem with `is_fbc` variable.. the definition is on `lower` level (spaces, tabs- my brain doesn't work) and then you use it on different

Reply:
ok, I'll fix that - but I guess it must be working since this is already on PROD :D

Reply:
fixed